src/extension.ts

```diff
@@ -82,7 +83,22 @@ export async function activate(context: vscode.ExtensionContext) {
 
   const graph = makeGraph(graphConstructors);
   var thatMark: SelectionWithEditor[] = [];
-
+  var recordTestCase = false;
+  const cursorlessRecordTestCaseDisposable = vscode.commands.registerCommand(
+    "cursorless.recordTestCase",
+    () => {
+      console.log("Recording test case for next command");
+      recordTestCase = true;
+      // Empty clipboard to avoid accidentally cluttering fixtures
+      vscode.env.clipboard.writeText("");
```

Comment:
See above comment; I'd argue we should just only grab the clipboard content in the case of a paste or copy